packages/cardano-services-client/scripts/createVersionSource.js

```diff
@@ -24,13 +24,24 @@ const apiVersion = Object.fromEntries(
 
 const versionFile = path.join(__dirname, '../version.json');
 const create = () => {
-  const version = JSON.stringify(apiVersion, null, 2);
+  let versionsData = {};
+  if (fs.existsSync(versionFile)) {
+    const data = fs.readFileSync(versionFile, { encoding: 'utf8' });
+    versionsData = JSON.parse(data);
+  }
+  for (const key of Object.keys(apiVersion)) {
+    if (!versionsData[key]) {
+      versionsData[key] = [apiVersion[key]];
+    } else if (!versionsData[key].includes(apiVersion[key])) {
+      versionsData[key].push(apiVersion[key]);
```

Comment:
If a major version is bumped, then this server doesn't support previous major versions. I suggest to also rename the json file to `supportedVersions.json`
```suggestion
      versionsData[key] = [...versionsData[key].filter(sameMajorVersion(apiVersion[key])), apiVersion[key]];
```

Reply:
Can we somehow fix up openapi definitions to just include versions that service supports?

Reply:
It's already doing that by using semantic version :)

Reply:
Fixed in 99e3c4d91fc6f7079a2509553b3e3977efd234d5